exporters/otlp/common/build.gradle.kts

```diff
@@ -15,6 +15,7 @@ dependencies {
     api(project(":sdk:all"))
     api(project(":sdk:metrics"))
 
+    api("org.curioswitch.curiostack:protobuf-jackson")
```

Comment:
do we really need to expose this as an `api` dependency?

Reply:
I moved `HexEncodingStringJsonGenerator` to `:exporters:otlp:common` because multiple classes depend on the ability to hex encode trace id and span id. That class contains a method `public static JsonGenerator create(...)` which returns a class `org.curioswitch.curiostack:protobuf-jackson`. 

So without `api` here, modules dependent on this code have to specify their own dependency on `org.cuioswitch.curiostack:protobuf-jackson`.

Reply:
Didn't look at the code yet but highest level comment is that I wasn't thinking of supporting json here. I don't know of any use case for it, as far as I know the main reason we have the protocol is for browsers where protobuf can cause challenges. Perhaps someone will ask for it later (I don't expect it TBH) but we can do it then.

Reply:
Well that simplifies a lot of stuff!